tests/src/FEManagerTest.cpp

```diff
@@ -46,12 +47,15 @@ TEST(DofHandler, DofHandlertest) {
 
   std::vector<Ikarus::FiniteElements::IFiniteElement> fes;
 
+  EXPECT_EQ(surfaces(gridView).size(), 2);
   for (auto&& ge : surfaces(gridView))
-    fes.emplace_back(Ikarus::FiniteElements::ElasticityFE(ge));
+    fes.emplace_back(Ikarus::FiniteElements::ElasticityFE(ge, gridView.indexSet()));
 
-  auto dh = Ikarus::DofManager::DefaultDofManager(fes, gridView);
+  auto dh = Ikarus::FEManager::DefaultFEManager(fes, gridView);
 
-  auto VariableList = dh.elementVariables()[0];
+  auto VariableList = dh.elementVariables()[0].get(Ikarus::EntityType::vertex);
+
+  EXPECT_THROW(dh.elementDofsVariableDataTuple(), std::logic_error);

```

Comment:
Um das Interface zu verstehen wäre es hilfreich, wenn es einen weiteren Test gibt, bei dem das funktioniert. Ansonsten sehe ich nur, dass es hier nicht geht, aber weiß weder warum noch was ich zurückbekomme wenn es in einem anderen Fall funktioniert

Reply:
diskussion